app/src/main/java/com/edulectronics/tinycircuit/Views/MenuActivity.java

```diff
@@ -14,28 +14,22 @@
 import java.util.Locale;
 
 public class MenuActivity extends AppCompatActivity {
-    String language ="nl";
+    String language = "nl";
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
                 WindowManager.LayoutParams.FLAG_FULLSCREEN);
         setContentView(R.layout.activity_menu);
+        //The split is used because language tags like 'en-US' and 'en-GB' should all use English
+        language = Locale.getDefault().toLanguageTag().split("-")[0];
```

Comment:
Aangezien we alleen 'nl' en 'en' hebben zou je ook StringUtils.substring(Locale.getDefault().toLanguageTag(), 0, 2) kunnen doen